helm/values.yaml

```diff
@@ -50,6 +50,17 @@ manager:
     python:
       repository: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-python
       tag: 0.43b0
+  autoAnnotateAutoInstrumentation:
+    java:
+      namespaces: [ ]
+      deployments: [ ]
+      daemonsets: [ ]
+      statefulsets: [ ]
+    python:
+      namespaces: [ ]
+      deployments: [ ]
+      daemonsets: [ ]
+      statefulsets: [ ]
```

Comment:
nit: Do we need to define these explicitly?

Reply:
I just kept these as a sort of self-documentation. If someone is just looking at our helm chart values.yaml, this makes it clear as to what options can be passed in.